settings.gradle.kts

```diff
@@ -14,3 +14,33 @@ sourceControl {
 }
 rootProject.name = "prime-reportstream"
 include("shared", "submissions", "prime-router", "auth")
+
+dependencyResolutionManagement {
+    versionCatalogs {
+        create("libs") {
+            version("hapi-structures_caffeine_client", "7.2.2")
```

Comment:
No objection to doing it this way, but I think there is some overlap here with what we're doing in the `buildSrc` and the reportstream plugin.

That seems like a slightly different pattern for keeping depedencies in sync so it might be good to decide on